main/src/cgeo/geocaching/connector/gc/GCLogin.java

```diff
@@ -74,15 +87,10 @@ protected StatusCode login(final boolean retry, @NonNull final Credentials crede
         }
 
         final String username = credentials.getUserName();
-        final String password = credentials.getPassword();
 
         setActualStatus(CgeoApplication.getInstance().getString(R.string.init_login_popup_working));
         try {
-            final Response tryLoggedInResponse = Network.getRequest("https://www.geocaching.com/login/default.aspx").blockingGet();
-            final String tryLoggedInData = Network.getResponseData(tryLoggedInResponse);
-            if (tryLoggedInResponse.code() == 503 && TextUtils.matches(tryLoggedInData, GCConstants.PATTERN_MAINTENANCE)) {
-                return StatusCode.MAINTENANCE;
-            }
+            final String tryLoggedInData = getLoginPage();
 
             if (StringUtils.isBlank(tryLoggedInData)) {
                 Log.e("Login.login: Failed to retrieve login page (1st)");
```

Comment:
Indeed, that would probably be wise, even though we haven't had identified problems with this old code path. I'll do it, thanks for the note.